src/pages/register.tsx

```diff
@@ -0,0 +1,210 @@
+import React from "react";
+import {
+  Box,
+  Typography,
+  TextField,
+  Stack,
+  Button,
+  SelectChangeEvent,
+  Grid,
+  InputAdornment,
+  FormHelperText,
+} from "@mui/material";
+import PasswordTextFeild from "@/components/public/PasswordTextField";
+import Logo from "@/components/public/Logo";
+import { Gender } from "enum/gender";
+import dayjs, { Dayjs } from "dayjs";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import CommonTextField from "@/components/public/CommonTextField";
+import CommonDropdown from "@/components/public/CommonDropdown";
+import { validation } from "@/types/Validation";
+import {
+  validateEmail,
+  validateTextField,
+  validatePasswordTextField,
+} from "@/utilities/validation";
+import { CHAR_LIMIT } from "enum/inputLimit";
+import { MobileDatePicker } from "@mui/x-date-pickers";
+import CalendarTodayIcon from "@mui/icons-material/CalendarToday";
+
+export default function Home() {
+  const [email, setEmail] = React.useState("");
+  const [password, setPassword] = React.useState("");
+  const [confirmPassword, setConfirmPassword] = React.useState("");
+  const [isSubmit, setIsSubmit] = React.useState(false);
+  const [isConfirm, setIsConfirm] = React.useState(true);
+  const [value, setValue] = React.useState<Dayjs | null>(null);
+  const [displayName, setDisplayName] = React.useState<string>("");
+  const [gender, setGender] = React.useState<string>("");
+  const [birthDate, setBirthDate] = React.useState<Dayjs | null>(dayjs());
+
+  const handleDisplayNameChange = (
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void => {
+    setDisplayName(event.target.value);
+    setIsSubmit(false);
+  };
+
+  function handleEmailChange(
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void {
+    setEmail(event.target.value);
+    setIsSubmit(false);
+  }
+
+  function handlePasswordChange(
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void {
+    setPassword(event.target.value);
+    setIsSubmit(false);
+  }
+
+  function handleConfirmPasswordChange(
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void {
+    setConfirmPassword(event.target.value);
+    setIsSubmit(false);
+  }
+
+  const handleSelectChange = (event: SelectChangeEvent): void => {
+    setGender(event.target.value as string);
+  };
+
+  const handleBirthDate = (newValue: Dayjs | null) => {
+    if (newValue && newValue.isAfter(dayjs())) {
+      console.error("Birth Date must be in the past.");
+    } else {
+      setBirthDate(newValue);
+    }
+  };
```

Comment:
I think all users may not open console to inspect the error, so lacking error response to user still lack.
Or would it much easier to disable the future date in the picker?